### DISCUSSION TITLE
征求多字少字纠错数据集

### DISCUSSION THREAD

**Discussion by shibing624** [shibing624/pycorrector#307] [Show and tell]
1、求一份高质量的带多字少字错误的纠错数据集，可以是比赛用的、自己标注的、数据集数量不限；
2、数据集出现的错误更希望接近人为书写的错误类型；
3、该数据集可以非商用，开源的，技术研究使用，如果是Apache 2.0协议也支持商用更好。

**Comment by YYTB** [comment]
这个进展如何了，请问。

**Comment by shibing624** [comment]
1. MuCGEC的训练集：训练数据，我们实验所用训练集为：Lang8数据集(来自外语学习网站Lang8）和HSK数据集(北语开发的汉语学习考试数据集)中的错误句子，并且对HSK数据集上采样5次，过滤掉和我们测试集重复的部分，共计约150万对。

下载方式：[Google Drive](https://drive.google.com/file/d/1l0A50z7fMXjQT3y2ct7TQsEHqOwlvg0_/view?usp=sharing)

2. MuCGEC数据集作为CCL2022-CLTC评测的Track 4在[阿里云天池平台](https://tianchi.aliyun.com/dataset/dataDetail?dataId=131328)开放， reference：https://github.com/HillZhang1999/MuCGEC

3. https://github.com/blcuicall/CCL2022-CLTC CCL 2022 汉语学习者文本纠错评测
